os_faults/drivers/tcpcloud.py

```diff
@@ -232,12 +232,6 @@ class SwiftAccountReplicatorService(SaltService):
     SALT_SERVICE = 'swift-account-replicator'
 
 
-class SwiftAccountReplicatorServerService(SaltService):
```

Comment:
Are you intending to revert all of these changes? (looks like quite a few in this commit)